content/Reducing_Power_Consumption.md

```diff
@@ -59,14 +59,21 @@ power standby mode. For example, a program may use this feature when it
 hasn't sensed keyboard input for a longer period (for example, when
 somebody forgot to turn off the Game Boy).
 
-Before invoking STOP, it might be required to disable Sound and Video
-manually (as well as IR-link port in CGB). Much like HALT, the STOP
-state is terminated by interrupt events. STOP is commonly terminated
-with a joypad interrupt.  A bug may keep STOP from terminating if STOP
-was invoked while a button was held.
-
-During STOP mode, the display will turn white, so avoid using it in your
-game's main loop.
+No licensed rom makes use of STOP outside of CGB speed switching.
+Special care need to be taken if you want to make use of the STOP
+instruction.
+
+On a DMG, disabling the LCD before invoking STOP, leaves the LCD enabled
+drawing a horizontal black line on the screen and damaging the hardware.
+
+On CGB, leaving the LCD enabled when invoking STOP will return in a
```

Comment:
Not sure about the "return in" there?

Reply:
my head remembers this as "results", so:
> ... will result in a ...